mscore/shortcut.cpp

```diff
@@ -114,17 +114,6 @@ Shortcut Shortcut::_sc[] = {
          Qt::WindowShortcut,
          ShortcutFlags::A_SCORE
          },
-      {
-         MsWidget::MAIN_WINDOW,
-         STATE_NORMAL | STATE_NOTE_ENTRY | STATE_EDIT | STATE_PLAY,
-         "file-part-export",
-         QT_TRANSLATE_NOOP("action","Export Parts…"),
-         QT_TRANSLATE_NOOP("action","Export parts"),
-         QT_TRANSLATE_NOOP("action","Save a copy of the score's parts in various formats"),
-         Icons::fileSave_ICON,
-         Qt::WindowShortcut,
-         ShortcutFlags::A_SCORE
-         },
```

Comment:
If you remove it from here, it means it won't be assignable to a shortcut anymore.

Reply:
Yes, that is intended. The idea is that there will be a single action for export, which opens the Export dialog, where you can select whether you want the score and/or the parts.
(That is the action above the deleted one, so you don't see it directly in the GitHub diff.)